src/pages/BaseResultsPage.js

```diff
@@ -3,98 +3,20 @@
 import React from "react";
 import { History } from "react-router";
 import reactMixin from "react-mixin";
-import _ from "underscore";
 
 import iss from "../iss";
 import BaseCategoriesPage from "./BaseCategoriesPage";
+import ResultsListPage from "./ResultsListPage";
+import ResultsMapPage from "./ResultsMapPage";
```

Comment:
You can still pass props. There's a few places we do things like this. `React.cloneComponentWithProps` will do it for instance.